app/templates/components/forms/wizard/attendee-step.hbs

```diff
@@ -9,15 +9,20 @@
               {{input type='checkbox' checked=data.event.isTicketFormEnabled}}
               <label class="weight-300" style="font-size: large">
                 {{if data.event.isTicketFormEnabled (t 'Turn off') (t 'Turn on')}}
-                {{t 'Ticket Form'}}
```

Comment:
@kushthedude a query. Why was it `Ticket Form` earlier? I guess I missed these changes.

Reply:
I don't know, Why was it named `Ticket Form`, But still we should correct our mistakes .